backend/libs/event-utils.ts

```diff
@@ -1,40 +1,40 @@
 import EventBridge from 'aws-sdk/clients/eventbridge';
-import { MessageFromServerToExtensionType } from '../../extension/src/communications/from-server-to-extension';
+import { MessageFromServerToExtension } from '../../extension/src/communications/from-server-to-extension';
 import { Room } from '../../extension/src/types';
 import { IApplicationEventWrapper, IEvent } from './response';
 
 const eventBridge = new EventBridge();
 
 export const buildEvent = (
-    type: MessageFromServerToExtensionType,
-    originEvent: IEvent,
+    message: MessageFromServerToExtension,
     room: Room,
+    originEvent: IEvent,
 ) => {
     const readEvent: IApplicationEventWrapper = {
-        type: type,
         requestContext: originEvent.requestContext,
-        data: room,
+        room,
+        message,
     };
 
     return {
         Source: process.env.EVENT_SOURCE,
         EventBusName: process.env.MEDIA_EVENT_BUS,
-        DetailType: type,
+        DetailType: message.type,
         Detail: JSON.stringify(readEvent),
     };
 };
 
 export const sendEvent = async (
     event: IEvent,
     room: Room,
-    type: MessageFromServerToExtensionType,
+    message: MessageFromServerToExtension,
 ) => {
-    const roomEvent = buildEvent(type, event, room);
-    console.log(roomEvent);
+    const eventForEventBridge = buildEvent(message, room, event);
+    console.log(eventForEventBridge);
```

Comment:
We should either add a debug message or remove this log

Reply:
We have proper logging for the extension, but regarding logging on AWS (Cloudwatch logs, we don't have yet alternatives to console.log(). I'll add a ticket on our github project board)

Reply:
I meant like adding some text so that when looking at the logs we have some context.

For example:  `console.log('event being sent to event bridge:', eventForEventBridge)`